tutorials/Working_with_fNIRS_data.py

```diff
@@ -0,0 +1,123 @@
+                      ['HbR %.2d' % i for i in range(1, n_channels + 1)]
+channel_names = channel_names_fnirs + ['Events']
+channel_types = ['hbo'] * n_channels + ['hbr'] * n_channels + ['stim']
+info = mne.create_info(ch_names=channel_names, sfreq=sfreq,
+                       ch_types=channel_types, montage=None)
+
+###############################################################################
+# Import fNIRS data from numpy array using :class:`mne.io.RawArray`.
+raw_data = mne.io.RawArray(data=raw_ndarray, info=info, first_samp=0,
+                           verbose=None)
+
+###############################################################################
+# In our data array the last row contains event data, coded as different
+# numbers in a time series synchronous with the fNIRS recording. Let's find
+# such events using :func:`mne.find_events` and plot the events timeline using
+# :func:`mne.viz.plot_events`.
+events = mne.find_events(raw_data, stim_channel='Events', shortest_event=1)
+event_id = {'ISI': 1, 'Biological': 2, 'Rotation': 4, 'Scrambled': 8,
+            'Pause': 16}
+color = {1: 'black', 2: 'green', 4: 'magenta', 8: 'cyan', 16: 'yellow'}
+mne.viz.plot_events(events, raw_data.info['sfreq'], raw_data.first_samp,
+                    color=color, event_id=event_id)
+
+###############################################################################
+# Sometimes we want to filter out unwanted frequencies from time series. For
+# example, here we bandpass filter our data between 0.01 and 2 Hz.
+l_freq = 0.01  # high-pass filter cutoff ( __/¯¯¯ )
+h_freq = 2     # low-pass filter cutoff ( ¯¯¯\__ )
+raw_data.filter(l_freq, h_freq, fir_design='firwin')
+
+###############################################################################
+# In order to have a general glimpse of the dataset, we start by plotting all
+# channels into a single plot window. We choose to color-code HbO, HbR and
+# events data using red, blue and black respectively. We also add color-coded
+# events positions, using the same colors previously used in
+# :func:`mne.viz.plot_events`.
+scalings = dict(hbo=10e-6, hbr=10e-6, stim=1)
+fig_title = 'fNIRS raw bandpass filtered %s Hz - %s Hz' % (l_freq, h_freq)
+plot_colors = dict(hbo='r', hbr='b', stim='k')
+raw_data.plot(title=fig_title, events=events, start=0.0, color=plot_colors,
+              event_color=color, duration=np.max(raw_data.times),
+              scalings=scalings, order='original',
+              n_channels=len(channel_names), remove_dc=False, highpass=None,
+              lowpass=None)
+
+###############################################################################
+# We can later dive into the dataset and look at the raw time series more
+# closely. Same color-codes through the plots is usually a good idea.
+fig_title = 'fNIRS raw bandpass filtered %s Hz - %s Hz' % (l_freq, h_freq)
+plot_colors = dict(hbo='r', hbr='b', stim='k')
```

Comment:
Please update these (and possibly the scalings) here:

https://github.com/mne-tools/mne-python/blob/master/mne/defaults.py#L13

We should probably explicitly pick some aesthetically nice red/blue from here instead of using 'r' and 'b' (which change from 1.+ -> 2.+ of mpl):

https://cloud.githubusercontent.com/assets/11820627/19903655/9ec8a21e-a02c-11e6-8778-80c2f2c068fb.png